app/models/profile.rb

```diff
@@ -4,25 +4,15 @@ class Profile < VersionedModel
   before_validation :set_assessment_answers
 
   belongs_to :person
-  belongs_to :ethnicity, optional: true
-  belongs_to :gender, optional: true
 
   has_one :move, dependent: :nullify
 
   has_many :documents, -> { kept }, as: :documentable, dependent: :destroy, inverse_of: :documentable
 
   validates :person, presence: true
-  validates :last_name, presence: true
-  validates :first_names, presence: true
+
   validate :validate_assessment_answers
   attribute :assessment_answers, Types::Jsonb.new(Profile::AssessmentAnswers)
-  attribute :profile_identifiers, Types::Jsonb.new(Profile::ProfileIdentifiers)
-
-  scope :ordered_by_name, ->(direction) { order('last_name' => direction, 'first_names' => direction) }
-
-  IDENTIFIER_TYPES = %w[
```

Comment:
Todo: triple check with Dom / Jason that `niche_reference` and `athena_reference` can be removed from the system, and check that they have no values on staging or on production.